docs/guides/testing.md

```diff
@@ -83,8 +83,9 @@ In the next steps we are going to setup our Jest environment in order to mock Zu
 
 ```ts
 // __mocks__/zustand.ts
-import * as zustand from 'zustand'
 import { act } from '@testing-library/react'
+import type * as zustand from 'zustand'
```

Comment:
Can you do the same for others please?

Reply:
In this case we use it to type the import from `requireActual` or

Reply:
@dai-shi keep me in the loop

Reply:
Oh, I see L91.

Can we do something like this? Feels a little more explicit.
```suggestion
import type * as ZustandExportedTypes from 'zustand'
```

Reply:
@dai-shi sure